src/App.js

```diff
@@ -183,7 +200,16 @@ function App() {
       </section>
       <section className="blue section">
       <div className="card">
-       <Magnifier src={visualizer} id="cnn-vis" mgShape="square" mgWidth={100} mgHeight={100} zoomFactor="1.5" mgShowOverflow="false" alt="two filters applied to one layer of a CNN"/>
+       <button id="fullscreen" onClick={handle.enter}>
+         View full image
+       </button>
+
+       <FullScreen handle={handle}>
+         <div className="cnnHolder">
+            <img src={visualizer} id="cnn-vis" alt="two filters applied to one layer of a CNN"/>
+         </div>
+       </FullScreen>
```

Comment:
I like that y'all are taking different approaches to making the image clearer, but I don't think this solves the problem: all it does is full-screen the app, but it doesn't zoom in on the image itself, which doesn't help with the clarity.

On top of that, I'd generally avoid forcing the user to switch into full-screen mode: other than weird browser shenanigans, it's a generally frowned upon approach and has some accessibility issues.

Reply:
Got it- the real plan for now is to split it up into smaller panels that they can click through, but we're trying to finish the earlier sections first so it might not happen for a while